app/views/users/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1><%= @user.name %> Profile </h1>
```

Comment:
The show page must also include events created by a user, clearly separated from past and upcoming events.

[`REQUIRED`]

Reply:
I am bit confused about this ? I have added upcoming events and past events which are created by that user only and invitations below that.
![Screen Shot 2019-09-18 at 12 56 31 PM](https://user-images.githubusercontent.com/25217715/65125934-f61f7000-da13-11e9-8609-b33a003d3da9.png)